app/assets/javascripts/offendersummary/components/offenderSummaryPage.js

```diff
@@ -15,6 +15,10 @@ class OffenderSummaryPage extends Component {
         getOffenderDetails();
     }
 
+    componentDidUpdate() {
+        window.GOVUKFrontend.initAll();
```

Comment:
maybe move this to the main html template? In theory `componentDidUpdate` could be called multiple times